src/kilt/temel.cr

```diff
@@ -0,0 +1,9 @@
+require "../kilt"
+require "temel"
+
+macro embed_temel(filename, __kilt_io__)
+  __kilt_io__ << {{ run("./helpers/temel_embedder.cr", filename) }}
```

Comment:
I think the path to the embedder needs to be as if it was being included in an external project like: "kilt/kilt/helpers/temel_embedder" or else it won't work when people require it. I had to do that in slang.

Also, this new path will fail the tests :( unless you add symlinks to make that path accessible.

Might be best to add it to temel directly.